### DISCUSSION TITLE
Fix: foreign languages channels in country list

### DISCUSSION THREAD

**Discussion by nomadmzl** [iptv-org/community#432] [Q&A]
### Describe your issue

Hi, may I suggest an edition in [Colombia channels list](https://iptv-org.github.io/iptv/countries/co.m3u)?

There are a lot of foreign language channels that barely would be watched, like hungarian, korean and japanese ones; I could do a small edition leaving only those in spanish language and tropical music if you want.

Bye.

**Comment by AntiPontifex** [comment]
Dead issue: please close.

**Comment by freearhey** [comment]
If a channel should not be in the playlist of some country, it is enough to fix `broadcast_area` of this channel in the [iptv-org/database](https://github.com/iptv-org/database) and at the next update it will disappear from it.

https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#how-to-edit-feed-description
